src/ocamlorg_frontend/layouts/package_layout.eml

```diff
@@ -48,29 +54,29 @@ Layout.render
       </div>
       <div class="flex flex-col">
         <div class="flex mt-8 gap-x-6 border-b border-gray-200 items-stretch flex-wrap">
-          <a href="<%s Url.package_with_version package.name package.version %>">
-            <button
-              class="<%s if tab = Overview then current_tab_class else tab_class %>">
-              Overview
-            </button>
-          </a>
-          <a href="<%s Url.package_doc package.name package.version "index.html" %>">
-            <button
-            class="<%s if tab = Documentation then current_tab_class else tab_class %>">
-              Documentation
-              <% (match documentation_status with 
-                 | `Failure -> %>
-                 <span class="bg-red-500 ml-2 absolute top-2 p-1 rounded-full inline-block">
-                 </span>
-              <% | `Success -> %>
-                <span class="bg-green-500 ml-2 absolute top-2 p-1 rounded-full inline-block">
-                </span>
-              <% | `Unknown -> %>
-                <span class="bg-orange-300 ml-2 absolute top-2 p-1 rounded-full inline-block">
-                </span>
-              <% ); %>
-            </button>
+          <a href="<%s Url.package_with_version package.name package.version %>" class="<%s if tab = Overview then current_tab_class else tab_class %>">
+            Overview
           </a>
+
+          <% (match documentation_status with
+            | `Success -> %>
+            <a href="<%s Url.package_doc package.name package.version "index.html" %>"
+              class="<%s if tab = Documentation then current_tab_class else tab_class %>">
+                Documentation
+                <%s! documentation_dot %>
+            </a>
+          <% | _ -> %>
+            <span class="<%s unavailable_tab_class %>"
+              title="<%s match documentation_status with
+                | `Failure -> "documentation failed to build"
```

Comment:
We might still want to have a link to the documentation if it's a failure: the page then displays the documentation error, which can be useful for package authors (and the documentation pipeline maintainers) to debug the issue

Reply:
Can you show me one such case? So far, all the ones I saw were 404, but that may be because I haven't come across the correct one yet.
The one you mean is in case we have a build failure? Then indeed, I'll change that!

Reply:
Only the `Unknown` documentation status is now a deactivated tab. Indeed, I hadn't considered that in case the docs server provides the build errors, we still need to link to that.